src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1121,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
```

Comment:
Good job make the function arguments readable.
Though it seems that the implementation is not what the T2A2 wants.
T2A2 requires only the prefix at the start of the string to be removed, this implementation does not fulfill the requirements.